README.md

```diff
@@ -1,254 +1,111 @@
 # note-zephyr
 
-A `west` compatible [Zephyr RTOS](https://zephyrproject.org/) module for the [Notecard](https://shop.blues.io/collections/notecard). Include this module in your own project manifest and use the `notecard` module to access the Notecard.
+This is a Zephyr `west` module for the Blues Notecard. It is an abstraction layer for the [note-c](https://github.com/blues/note-c) library, that allows you to easily use a [Notecard](https://blues.com/products/notecard/) in a Zephyr application.
 
-## Overview
-
-This example is designed to demonstrate the ease of adding Notecard functionality
-to an existing Zephyr application.
-To quickly get started, use the `blinky` example, check out the `examples/blinky` project.
+It supports both `I2C` and `UART` communication with the Notecard as well as utilizing Zephyr native APIs for logging and error handling as well as memory management.
 
 ## Requirements
 
-To use this module, we make the following assumptions to make it easier to get started:
+While this module is designed to be used with the Blues' Feather MCU boards, it can be used with any Zephyr compatible board. Overlays may need to be modified to work with other board device trees.
 
 ### Hardware
 
 - Any [Blues Notecard](https://shop.blues.io/collections/notecard)
 - [Blues Notecarrier F](https://shop.blues.com/products/notecarrier-f)
-- [Blues Swan](https://shop.blues.io/products/swan)
+- Blues Feather MCU boards
+  - [Swan](https://shop.blues.io/products/swan)
 - [STLINK Programmer/Debugger](https://shop.blues.io/collections/accessories/products/stlink-v3mini)
 
-### Software
-
-- [Docker](https://docs.docker.com/get-docker/)
-- [VSCode](https://code.visualstudio.com/Download)
-- [VSCode "Dev Containers" Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
-- **\[Windows/Mac Debugging\]** [OpenOCD](https://openocd.org/pages/getting-openocd.html)
-
-### Cloudware
+### Software / Cloudware
```

Comment:
I don't immediately see the advantage of native development over dev containers.

Native development has only ever been a support nightmare, especially with Zephyr (who constantly change requirements, and introduce breaking changes to the build system).

Reply:
This changes now with zephyr modules (as we discussed) as there's no need for us to worry about the user's environment, they import the module with west and build against that. Added this comment for context from out discussion.